services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/outputs/_watchdog_extensions.py

```diff
@@ -21,6 +24,8 @@
     ],
 )
 
+logger = logging.getLogger(__name__)
+
 
 class _ExtendedInotifyBuffer(InotifyBuffer):
     def __init__(self, path, recursive=False):  # pylint:disable=super-init-not-called
```

Comment:
Please add a note explaining why you call ``BaseThread`` and not ``InotifyBuffer``. (I remember having this discussion but do not remember the conclusions)